client/src/charts/legends/LegendList.tsx

```diff
@@ -2,20 +2,34 @@ import classNames from "classnames";
 import _ from "lodash";
 import React from "react";
 
-import { CheckBox } from "src/components/CheckBox/CheckBox.js";
+import { CheckBox, CheckBoxProps } from "src/components/CheckBox/CheckBox";
 
 import { is_a11y_mode } from "src/core/injected_build_constants.js";
 
 import "./LegendList.scss";
 
+export interface LegendItemProps {
+  active: boolean;
+  id: string | number;
+  label: string;
+  color?: string;
```

Comment:
Might make a separate ts file for all legends to use. Will take care of all legends and include that change later